setup.py

```diff
@@ -29,7 +29,7 @@
     long_description=long_description,
     long_description_content_type="text/plain",
     packages=find_packages(),
-    version="4.0.0",
+    version="4.0.1",
```

Comment:
We will have to check whether this will end up in PyPI automatically. Last time when we bumped the version in GitHub, it magically ended up in PyPI, although we wanted to release it later. We thought that releasing a package to PyPI involves a few more steps. We'll see this time...